.eslintrc.cjs

```diff
@@ -0,0 +1,8 @@
+module.exports = {
+  extends: ['@massalabs'],
+  rules: {
+    'tsdoc/syntax': 'warn',
+    'max-len': ['error', 200],
+    camelcase: 'off',
+  },
```

Comment:
because it's a brand new project, we must follow our rules from the beginning. If our rules in https://github.com/massalabs/eslint-config let's open an issue there

```suggestion
```

Reply:
I have fixed the code elsewhere and removed these rules.